src/app.py

```diff
@@ -58,38 +63,24 @@
 # Suppress InsecureRequestWarning warning when requesting status on https with ssl cert verify disabled
 requests.packages.urllib3.disable_warnings(category = InsecureRequestWarning)
 
+
 ####################################################################################################
-## Register error handlers
+## UBKG Ontology and REST initialization
 ####################################################################################################
 
-# Error handler for 400 Bad Request with custom error message
-@app.errorhandler(400)
-def http_bad_request(e):
-    return jsonify(error = str(e)), 400
-
-
-# Error handler for 401 Unauthorized with custom error message
-@app.errorhandler(401)
-def http_unauthorized(e):
-    return jsonify(error = str(e)), 401
-
-
-# Error handler for 403 Forbidden with custom error message
-@app.errorhandler(403)
-def http_forbidden(e):
-    return jsonify(error = str(e)), 403
-
-
-# Error handler for 404 Not Found with custom error message
-@app.errorhandler(404)
-def http_not_found(e):
-    return jsonify(error = str(e)), 404
-
-
-# Error handler for 500 Internal Server Error with custom error message
-@app.errorhandler(500)
-def http_internal_server_error(e):
-    return jsonify(error = str(e)), 500
+try:
+    for exception in get_http_exceptions_classes():
+        app.register_error_handler(exception, abort_err_handler)
+    app.ubkg = initialize_ubkg(app.config)
```

Comment:
@maxsibilla I'm not sure how you handle the UBKG initialization details in the atlas-consortia-commons package, it's a good idea of having some error handling to notify any misconfiguration. Or add some validation/check here to ensure all the required config keys are present.

Reply:
In the Atlas Consortia Commons we have defined these standard error handlers:
https://github.com/x-atlas-consortia/commons/blob/main/atlas_consortia_commons/rest/__init__.py

Reply:
I was not referring to the flask error handlers. 

`app.ubkg = initialize_ubkg(app.config)` takes the config object as input instead of explicitly specifying the keys, we want some validation/check to the existence of the following items and the values:
```
UBKG_SERVER =
UBKG_ENDPOINT_VALUESET =
UBKG_CODES =
```

Reply:
Oh I understand. There is validation in the Atlas Consortia Commons package specifically for UBKG when `initialize_ubkg` gets called but I added explicit handling for missing parameters here: https://github.com/hubmapconsortium/files-api/pull/37/commits/c2eec94b892ad3232d8ad263a76f42d963d47a7b

Reply:
Thanks @maxsibilla !

Reply:
FYI This should work for hubmap:
```
#UBKG_SERVER = 'https://ontology-api.dev.hubmapconsortium.org/'
#UBKG_ENDPOINT_VALUESET = 'valueset?parent_sab=HUBMAP&parent_code={code}&child_sabs=HUBMAP'
#UBKG_CODES = '{"assay_types":{"code": "C004000", "key": "datasets", "endpoint": "datasets?application_context=HUBMAP"}, "organ_types":{"code": "C000008", "key": "organs", "endpoint": "organs?application_context=HUBMAP"}}'
```